### PR TITLE
Alternate local docker

### PR DESCRIPTION
**Pull request by ldimaggi**
Added a new script to be used to run tests locally in docker

### CODE REVIEW

**Comment by fabric8cd** [comment]
@ldimaggi fabric8/fabric8-ui:SNAPSHOT-PR-2080-1 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2080-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@ldimaggi fabric8/fabric8-ui:SNAPSHOT-PR-2080-2 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2080-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@ldimaggi fabric8/fabric8-ui:SNAPSHOT-PR-2080-3 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2080-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@ldimaggi fabric8/fabric8-ui:SNAPSHOT-PR-2080-5 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2080-fabric8-planner.badger.fabric8.io

File: scripts/local_docker2.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -x
+
+docker build -t fabric8-planner-builder -f deploy/Dockerfile.builder .
+mkdir -p runtime/dist && docker run --detach=true --name=fabric8-planner-builder --user=root --cap-add=SYS_ADMIN -t -v $(pwd)/runtime/dist:/dist:Z fabric8-planner-builder
```
Comment (hectorj2f) [inline scripts/local_docker2.sh]: wasn't it better to have a single local_docker.sh file and just add a bash param whenever we want to execute these commands or the existing ones :( ?

**Pull request merged**